### DISCUSSION TITLE
The T114 V2.0 is out and fixes the hardware issue so can the FW be placed back on the web flasher now?

### DISCUSSION THREAD

**Discussion by captainchaosrules** [meshtastic/Discussions#39] [Firmware]
https://heltec.org/project/mesh-node-t114/

The V2.0 fixes this issue so hopefully you can put the FW back on the Web Flasher soon - thanks! 
https://www.reddit.com/r/meshtastic/comments/1fnqq09/temporarily_removing_heltec_t114_from_official/?share_id=zjqWqfqQttCycB8vr1i4b&utm_content=2&utm_medium=ios_app&utm_name=ioscss&utm_source=share&utm_term=1

**Comment by thebentern** [comment]
Already been back in there for a few days :-)

**Comment by martinbogo** [comment]
I'm sad to report that all four of my t114 "V 2.0's" are showing many of the same symptoms as before.  Most concerning is that on 2.5.12->2.5.15 they all have this tendency to reboot randomly, bootloop, and that the displays aren't always coming on when booting.  

Since this is across four different devices, with and without batteries, on four different power supplies -- I'm fairly certain at this point that this is again a hardware related issue.   

I'll sit back and see how many other people are having issues with "V2" hardware...

**Reply by rotorHead82** [reply]
Just got my V2 replacement. Will update if mine is working after I have had time to play with it.

**Comment by itssunnyb** [comment]
is this still an issue, or has this been resolved? I don't see the T114 in the web flasher.
